.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
   - curl ifconfig.co|xargs echo "Travis IP address is ";
 
 script:
-  - "mvn cobertura:cobertura"
+  - "mvn fmt:check cobertura:cobertura"
   - mvn test -B
```

Comment:
Not that it needs to be done here but this line is redundant with cobertura:cobertura